integrations/destinations/apache-iceberg.mdx

```diff
@@ -257,6 +257,19 @@ WITH (
 
 Currently, RisingWave only supports Iceberg tables in format v2.
 
+## Exactly-once delivery
+
+Exactly-once semantics ensures that each record is processed and written exactly once, even in the event of system failures or retries.
+
+For Iceberg sinks, this is achieved through a two-phase commit protocol:
```

Comment:
I don't think we need to expose the implementation detail to users, instead we need to tell them what is "exactly once".